US-CBP/cbp-theme
# feat: initial style-dictionary implementation for design tokens

#### What's this PR do?
- Added `style-dictionary` package and `config.js` file added. Previous color tokens replaced by ones generated from `ds-css-v2/styles/tokens`.
- Folder structure changed in `ds-css-v2/styles/sass` with the addition of `/base` folder used for global styling and `/utilities` for any helper `scss`. 
- All CBP `scss` imports moved from `ds-css-v2/styles/sass/_uswds-theme-custom-styles.scss` to `ds-css-v2/styles/sass/styles.scss` 
#### Where should the reviewer start?
- `ds-css-v2/config.js`
- `ds-css-v2/styles/tokens`
- `ds-css-v2/styles/sass/cbp-tokens`
- `ds-css-v2/styles/sass/styles.scss` 
#### How should this be manually tested?
- Pull branch and run `$ npm install`
- Make sure the command `$ npm run style-dictionary` runs successfully
- Check that the tokens work in scss files. 
#### Any background context you want to provide?
- The purpose of this PR is to centralize color tokens in the project so that it can be handled by `style-dictionary` and use of `JSON` files.
#### What are the relevant tickets?
- JIRA
#### Screenshots (if appropriate)
- n/a
#### Questions:
- Is there a blog post?
- Does the knowledge base need an update?
